src/analysis/enums.rs

```diff
@@ -117,6 +117,14 @@ pub fn new(env: &Env, obj: &GObject, imports: &mut Imports) -> Option<Info> {
         }
     }
 
+    // Enum to_string functions provide static strings and need special error handling.
+    if let Some(_to_string) = functions.iter_mut().find(|f| f.name == "to_string") {
```

Comment:
This is only valid if the C function's return value is `transfer none`

Reply:
Agreed. I didn't add too much upfront checking in this proof of concept, all `to_string`s (in gstreamer) have that.